core/domain/exp_jobs_one_off_test.py

```diff
@@ -1182,22 +1191,25 @@ def test_for_textangular_validation_and_migration(self):
             self.assertEqual(
                 updated_html, unicode(test_cases[index]['expected_output']))
 
-        exp_services.save_new_exploration(
-            self.albert_id, updated_exploration)
+        with self.swap(
```

Comment:
Can you wrap this self.swap() only around the bit that is absolutely necessary?

Reply:
Done